Source/AssetDatabase/AssetHelper.cpp

```diff
@@ -48,6 +48,24 @@ namespace PlatinumEngine
 		return {isAssetSelected, mesh};
 	}
 
+	//TODO: Maybe a more efficient procedure to utilise TryGetAsset using filePath
+	std::tuple<bool, Mesh*> AssetHelper::GetMeshAsset(std::string filePath)
+	{
+		bool isAssetSelected = true;
```

Comment:
Is this guaranteed to be true? I feel like this should initially be false and then set to true if the filenames are the same where mesh is set